start-android.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+mkdir -p android/app/src/main/assets
+node node_modules/react-native/local-cli/cli.js bundle \
+	--platform android --dev false \
+	--entry-file index.js \
+	--bundle-output android/app/src/main/assets/index.android.bundle \
+	--assets-dest android/app/src/main/res
+adb uninstall com.agritrader
```

Comment:
why does it need to uninstall? Should we add a comment here since at first glance, it looks kind of suspicious?

Reply:
This is kind of a TODO, Every time an APK is installed onto a device it uses a key pair. That is generated per device.Since a new docker container is used every time, then the APK gets installed with a different signature. Android sees it as as security risk if an APK is being overwritten with a different keypair and it will fail. So the easy solution is just to uninstall the old one. 

The better solution would be to use a pregenerated keypair so the APK gets generated with the same keypair everytime. We probably need to do this in the future anyways when we generate deployable APKs. 

This is a long af comment for something that should change in the future, so I didn't want to comment about it in the file.

Reply:
ahh okay. Do you mind creating an issue for this then so we can keep track of it? :)